tests/kafkatest/services/streams.py

```diff
@@ -215,8 +216,11 @@ def node(self):
 
     def pids(self, node):
         try:
-            return [pid for pid in node.account.ssh_capture("cat " + self.PID_FILE, callback=int)]
-        except:
+            pids = [pid for pid in node.account.ssh_capture("cat " + self.PID_FILE, callback=str)]
+            return [int(pid) for pid in pids]
```

Comment:
I need to do this because otherwise the callback `int` will throw an exception when an error occured in `ssh_capture()` that returned an error message.